frontend/src/app/organization/organization-contact/organization-contact.component.html

```diff
@@ -61,7 +61,9 @@ <h2>Formularz kontaktowy</h2>
           required
           minlength="9"
           maxlength="15"
-          #phoneNo="ngModel" />
+          #phoneNo="ngModel"
+          [imask]="{mask: '+48 000-000-000'}"
+          [unmask]="true"/>
```

Comment:
It seems that validation of this field resolves when the focus is changed, but from US point of view, it is better when error alert disappear immediately when a correct value is provided. I think adding (complete)="onComplete" (just like it is shown in angular-imask documentation) can solve this problem.

Reply:
I will fix it. Do you think it would be better to add this to the # 1064 pullrequest again, or should I add it in addition, to another pullrequest, or make some completely new pullrequest only for this change? What is a good practice?